Task_2_1_1/build/classes/java/main/META-INF/BenchmarkList

```diff
@@ -0,0 +1,3 @@
+JMH S 31 r.masud.ParallelStreamBenchmark S 70 r.masud.jmh_generated.ParallelStreamBenchmark_containsNonPrime_jmhTest S 16 containsNonPrime S 11 AverageTime E A 1 1 1 E E E E E E E E E E E E E M 1 4 size 3 16 xAAMAADAwAA===== 16 xAAMAADAwAAMAA== 16 xAAMAADAwAAMAADA U 12 MILLISECONDS E E 
```

Comment:
build directory should be also removed from index

You can ask your groupmates for help with gitignore

Reply:
okay )